packages/mui-material/src/Rating/Rating.test.js

```diff
@@ -225,6 +225,12 @@ describe('<Rating />', () => {
       expect(screen.getByRole('img')).toHaveAccessibleName('Stars: 2');
     });
 
+    it('can be labelled with getLabelText when non set value', () => {
+      render(<Rating readOnly />);
+
+      expect(screen.getByRole('img')).toHaveAccessibleName('0 Heart');
```

Comment:
It should be in plural.

```suggestion
      expect(screen.getByRole('img')).toHaveAccessibleName('0 Hearts');
```